UPGRADE-2.0.md

```diff
@@ -1,5 +1,38 @@
 # Upgrade from 1.10 to 2.0
 
+Version `2.x` is a complete rewrite of the existing code base. The public API has been trimmed down to a minimum.
+The preferred way of using the SDK is through our static API.
```

Comment:
Not really `static API` but more like `(global) helpers` (better name possibly!? also their not really global...) as in the example below you using no `::` (except for some "enum" cases) which would indicate anything static method calls.

Reply:
Updated the phrasing there, I want to keep the "Static API" since it's described like that on the docs page, linked later.